lang/en.json

```diff
@@ -42,7 +42,7 @@
         "Start": "Start",
         "None": "None",
         "EndDate": "End Date",
-        "EndDateHelp": "The date to end importing at. By default, it will run every day until you decide to cancel it.",
+        "EndDateHelp": "The date to end importing at. By default, it will run every day until you decide to cancel it. It must be a valid date format (YYYY-MM-DD)",
```

Comment:
@snake14 Can we add the same for start_date too, since when you type placeholders are gone and it will be easy to lookup, and do you think we should make the format `YYYY-MM-DD` as bold ?

Reply:
@AltamashShaikh Sure. I wouldn't hurt to put it in both places. I like the idea of making the format stand out more, but I think it would be odd with the parenthesis. Should I remove them and bold the format?

Reply:
@Javi-Ormaechea What do you think? Right now, I've got it looking like this:
![image](https://user-images.githubusercontent.com/11394099/194990012-9cc6af62-4366-49c8-9de0-bc09a407bcdf.png)

So, I'll be adding the `It must be a valid date format (YYYY-MM-DD)` sentence to the end of the help text for the Start Date as well. I'm torn about bolding the `(YYYY-MM-DD)` date format. I like the idea of making it stand out more, but I think making the parenthesis bold would look odd as well as not making them bold while the date format is bolded. Should I remove the parenthesis, not bold the date format, bold just the date format, or bold the format and parenthesis?

Reply:
@snake14 Adding the format is good practice, so that looks good to me. No need to bold it.

I would suggest removing "It must be a valid date format" as you are already adding the format and that's the guidance our users need. 

"Please enter a valid date format (YYYY-MM-DD)" -> should go on the error message.

As an optimisation of this fields, it would be easier to have a date picker (might not work for this ticket but maybe as a next step)